src/main/java/ro/utcn/sd/mdantonio/StackUnderflow/entities/Post.java

```diff
@@ -0,0 +1,48 @@
+package ro.utcn.sd.mdantonio.StackUnderflow.entities;
+
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+import javax.persistence.*;
+import java.util.Date;
+
+@Data
+@Entity
+@NoArgsConstructor
+@Table(name="post")
+public class Post {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Integer postid;
+    private Integer posttypeid;
```

Comment:
I would prefer if you would use real associations between objects instead of mapping foreign keys as fields (i.e. have the Author object referenced inside the Post instead of having the authorId).